ADAL/src/public/ADUserInformation.h

```diff
@@ -34,15 +34,27 @@
 }
 
 /*! Factory method to extract user information from the AAD id_token parameter.
- @param idToken The contents of the id_token parameter, as passed by the server. */
-+ (ADUserInformation *) userInformationWithIdToken:(NSString *)idToken
-                                             error:(ADAuthenticationError * __autoreleasing *)error;
+ @param idToken The contents of the id_token parameter, as passed by the server.
+ */
++ (ADUserInformation *)userInformationWithIdToken:(NSString *)idToken
+                                            error:(ADAuthenticationError * __autoreleasing *)error;
+
+/*! Factory method to extract user information from the AAD id_token parameter.
+ @param idToken The contents of the id_token parameter, as passed by the server.
+ @param homeUserId Unique AAD user identifier across tenants based on user's home OID/home tenantId.
+ */
++ (ADUserInformation *)userInformationWithIdToken:(NSString *)idToken
```

Comment:
I'd suggest making this method private. Otherwise, if we want to remove it in future, we would still need to maintain it.

Reply:
done